test/bats/base-test.bats

```diff
@@ -125,6 +125,26 @@ RATIFY_NAMESPACE=gatekeeper-system
     assert_success
 }
 
+@test "notation test crl" {
```

Comment:
hi Juncheng, would you be able to add more detail in the description , for example , " notation verification pass on empty crl" to indicate what are we validating, and if this is a positive or negative test. thanks!

Reply:
Sure thing.